src/main/java/com/artipie/api/artifactory/GetUserSlice.java

```diff
@@ -70,20 +71,23 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
                         user -> {
                             final Response resp;
                             if (user.isPresent()) {
-                                resp = new RsJson(
-                                    () -> Json.createObjectBuilder()
-                                        .add("name", user.get().name())
-                                        .add(
-                                            "email",
-                                            user.get().email().orElse(
-                                                String.format("%s@artipie.com", user.get().name())
-                                            )
+                                final JsonObjectBuilder json = Json.createObjectBuilder()
+                                    .add("name", user.get().name())
+                                    .add(
+                                        "email",
+                                        user.get().email().orElse(
+                                            String.format("%s@artipie.com", user.get().name())
                                         )
-                                        .add("lastLoggedIn", "2020-01-01T01:01:01.000+01:00")
-                                        .add("realm", "Internal")
-                                        .build(),
-                                    StandardCharsets.UTF_8
-                                );
+                                    )
+                                    .add("lastLoggedIn", "2020-01-01T01:01:01.000+01:00")
+                                    .add("realm", "Internal");
+                                if (!user.get().groups().isEmpty()) {
```

Comment:
@olenagerasimova `groups` field marked as optional in docs, but I believe it means that it is optional when sent to server. It would be better to have it in `GET` response even if empty